products/compute/api.yaml

```diff
@@ -7358,6 +7358,38 @@ objects:
           The total number of nodes in the node group.
         required: true
         send_empty_value: true
+      - !ruby/object:Api::Type::NestedObject
+        name: 'autoscalingPolicy'
+        min_version: beta
+        description: |
+          If you use sole-tenant nodes for your workloads, you can use the node
+          group autoscaler to automatically manage the sizes of your node groups.
+        properties:
+          - !ruby/object:Api::Type::Enum
+            name: 'mode'
+            required: true
+            description: |
+              The autoscaling mode. Set to one of the following:
+                - OFF: Disables the autoscaler.
+                - ON: Enables scaling in and scaling out.
+                - ONLY_SCALE_OUT: Enables only scaling out. 
+                You must use this mode if your node groups are configured to 
+                restart their hosted VMs on minimal servers.
+            values:
+              - :OFF
+              - :ON
+              - :ONLY_SCALE_OUT
+          - !ruby/object:Api::Type::Integer
+            name: 'minNodes'
+            description: |
+              Minimum size of the node group, and must be an integer value less 
```

Comment:
This is a little awkwardly worded. Maybe "Minimum size of the node group. Must be an ..."